src/test.rs

```diff
@@ -90,18 +90,20 @@ use std::pin::Pin;
 #[cfg(feature = "websocket")]
 use std::task::{self, Poll};
 
-use async_stream::stream;
 use bytes::Bytes;
-use futures::{future, FutureExt, StreamExt, TryFutureExt};
+#[cfg(feature = "websocket")]
+use futures::StreamExt;
+use futures::{future, FutureExt, TryFutureExt};
 use http::{
     header::{HeaderName, HeaderValue},
     Response,
 };
 use serde::Serialize;
 use serde_json;
-use tokio::sync::mpsc::{self, UnboundedSender};
 #[cfg(feature = "websocket")]
-use tokio::sync::oneshot;
+use tokio::sync::{mpsc, oneshot};
+#[cfg(feature = "websocket")]
+use tokio_stream::wrappers::UnboundedReceiverStream;
```

Comment:
Does Rust support making both these `use`s dependent on the websocket feature, so you don't have to declare it for each?

Reply:
I don't think so

Reply:
OK thanks :+1: